.github/workflows/deploy-standalone-plugins.yml

```diff
@@ -0,0 +1,59 @@
+name: Deploy standalone plugins to WordPress.org
+
+on:
+  push:
+    branches:
+      - trunk
+      - 'release/**'
+    paths:
+      - '.github/workflows/deploy-standalone-plugins.yml'
+  pull_request:
+    branches:
+      - trunk
+      - 'release/**'
+      - 'feature/**'
+    paths:
+      - '.github/workflows/deploy-standalone-plugins.yml'
+    types:
+      - opened
+      - reopened
+      - synchronize
```

Comment:
I assume you have these here for testing, but I think we'll need to change this trigger to something different like a release, or the addition of a tag, or something. What do you think?

Reply:
This makes sense.

Performance Lab uses:
```yaml
on:
  release:
    types:
      - published
```

WordPress Beta Tester uses:
```yaml
on:
  push:
    tags:
    - "*"
```

Reply:
Yes @joemcgill, Once we have thoroughly tested the workflow, we must update it with a `release` or push `tags`. 

Reply:
@mukeshpanchal27 Can you please add both versions of the code here so that we can proceed with the full code review? You can comment out the version which we should eventually use (on release or manual dispatch), and then add a `TODO` comment to get rid of the current triggers here before we merge this.